src/backend/tests/unmocked/organization.test.ts

```diff
@@ -206,4 +206,26 @@ describe('Organization Tests', () => {
       expect(images.exploreAsGuestImage).toBe('uploaded-image2.png');
     });
   });
+
+  describe('Update Application Link', () => {
+    it('Fails if organization does not exist', async () => {
```

Comment:
You don't need this test anymore, instead replace it with fails if the user is not an admin